docs/paying-for-lit/overview.md

```diff
@@ -0,0 +1,129 @@
+# Paying for Usage of Lit
+
+The Lit networks, like other decentralized networks, manage their computational resources to ensure optimal performance and synchronization across their nodes. To maintain a responsive and balanced network, Lit implements a metering system for resource utilization.
+
+When using the Lit networks, whether it's the decentralized testnet ([Datil-test](../connecting-to-a-lit-network/testnets#datil-test)) or the production-ready mainnet beta ([Datil](../connecting-to-a-lit-network/mainnets#datil)) - you'll need to pay for and reserve your usage. This is accomplished using *Lit test tokens* and *Capacity Credits*.
```

Comment:
should we leave out the mention of test tokens here? As the credits themselves are how the capacity is reserved and the tokens are just a means of paying for the credits (which could be abstracted for the sake of simplicity imo)

Reply:
Changed to:

When using the Lit networks - whether it's the decentralized testnet ([Datil-test](../connecting-to-a-lit-network/testnets#datil-test)) or the production-ready mainnet beta ([Datil](../connecting-to-a-lit-network/mainnets#datil)) - you need to pay for and reserve your usage. This is accomplished using *Capacity Credits*.